### PR TITLE
Python: include cleanup, move casadi function generation in one file

### PR DESCRIPTION
**Pull request by FreyJo**

### CODE REVIEW

File: examples/acados_python/rsm_example/main.py [hunk 1]
```diff
@@ -208,7 +210,7 @@ def get_general_constraints_DC(u_max):
 nx = model.x.size()[0]
 nu = model.u.size()[0]
 nz = model.z.size()[0]
-np = model.p.size()[0]
+# np = model.p.size()[0]
```
Comment (sandmaennchen) [inline examples/acados_python/rsm_example/main.py:213]: just remove this line?

**Review by sandmaennchen — CHANGES_REQUESTED** [review]
could you order the functions in `casadi_function_generation.py`? :upside_down_face:

File: examples/acados_python/rsm_example/main.py (hunk 1 above)
Comment (FreyJo) [reply examples/acados_python/rsm_example/main.py]: Done!

**Comment by FreyJo** [comment]
All addressed!

**Review by sandmaennchen — APPROVED** [review]
:+1:

**Pull request merged**